src/Altinn.Common/Altinn.Common.AccessToken/Altinn.Common.AccessToken/Configuration/AccessTokenSettings.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Altinn.Common.AccessToken.Configuration
+{
+    public class AccessTokenSettings
+    {
+        public bool DisableAccesTokenVerification { get; set;  }
```

Comment:
I prefer positive names on fields, prioerties etc. Using "true" to enable something is more intuitive than using "true" to disable something. It is also easier to read logical expressions as it avoids double negatives. NotDisabled vs NotEnabled.

Reply:
The reason for this is that it by default should be true if this config is missing.

Reply:
That's a fair reason. It adds an implicit support for the default value of a boolean.
I just now also discovered that Acces needs an other s.